rpcs3/Emu/RSX/NV47/HW/common.cpp

```diff
@@ -62,8 +62,13 @@ namespace rsx
 			return vm::cast(get_address(offset, location));
 		}
 
-		void set_fragment_texture_dirty_bit(rsx::context* ctx, u32 index)
+		void set_fragment_texture_dirty_bit(rsx::context* ctx, u32 arg, u32 index, bool /*is_shader_config*/)
 		{
+			if (REGS(ctx)->latch == arg)

```

Comment:
Caller should check this on their side.

Reply:
This was to avoid including rsx_methods.h here.